.github/workflows/build-and-test-antioch.yml

```diff
@@ -0,0 +1,41 @@
+name: build-and-test-antioch
+run-name: Building And Testing Antioch
+on: [push]
+
+jobs:
+    build-on-windows:
+        runs-on: windows-latest
+        steps:
+          - name: Checkout Repo
+            uses: actions/checkout@v3
+
+          - name: Get CMake
+            uses: lukka/get-cmake@latest
+
+          - run: mkdir build
+          - run: cd build
+
+          - name: Set up Ninja Debug build
+            run: cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -S . -B build
+          - name: Build with Ninja
+            run: cmake --build build
+
+    build-on-linux:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Checkout Repo
+            uses: actions/checkout@v3
+
+          - name: Get CMake
+            uses: lukka/get-cmake@latest
```

Comment:
is this necessary?

Reply:
Yeah I believe so it needs to get cmake

Reply:
take a look at the github actions i wrote in GraphStreamingProject/GraphZeppelin. IIRC we don’t need to get cmake for ubuntu